bundle/manifests/policy-controller-operator.clusterserviceversion.yaml

```diff
@@ -227,13 +233,13 @@ spec:
           - subjectaccessreviews
           verbs:
           - create
-        serviceAccountName: policy-controller-operator-controller-manager
+        serviceAccountName: policy-op-controller-manager
```

Comment:
Where is the `op` in the name coming from? :) I read the permission matching comment from AI bot, but is that still a useful/meaningful name?

Reply:
op = operator
I was trying to shorten it, as the names of some of the resources where a bit long, I am happy to change it to something else, if you have any ideas :)